src/input.rs

```diff
@@ -697,6 +703,10 @@ impl State {
     }
 
     fn on_pointer_motion<I: InputBackend>(&mut self, event: I::PointerMotionEvent) {
+        if self.niri.tablet_mode {
```

Comment:
I'm thinking, maybe handle all of these one level above? I.e. before dispatching in `process_input_event()`, do:
```
if self.niri.tablet_mode && matches!(event, ...) {
    return;
}
```